contracts/modules/SuperMinterV1_1.sol

```diff
@@ -308,15 +335,13 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
 
         /* ------------------- CHECKS AND UPDATES ------------------- */
 
-        // Check if the mint is open.
-        if (LibOps.or(block.timestamp < d.startTime, block.timestamp > d.endTime))
-            revert MintNotOpen(block.timestamp, d.startTime, d.endTime);
-        if (_isPaused(d)) revert MintPaused(); // Check if the mint is not paused.
+        _requireMintOpen(d);
 
         // Perform the sub workflows depending on the mint mode.
         uint8 mode = d.mode;
         if (mode == VERIFY_MERKLE) _verifyMerkle(d, p);
```

Comment:
does typescript have switch case with way to error at compile time if a mode isn't handled?

Reply:
Regular Solidity does not have switch cases.

Reply:
😢